site/docs/types/1/0110-Actors.md

```diff
@@ -0,0 +1,48 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the Egeria project. -->
+
+# 0110 Actors
+
+Most metadata repositories are run in a secure mode
+requiring incoming requests to include the requester’s
+security credentials.
+Therefore we have an identifier for each unique logged on
+security identity (aka userId).
+This identity is recorded with specific entities and
+relationships when they are created or updated.
+The userId for a server is also captured in the metadata
+model (see [0040 Servers in Area 0](0040-Software-Servers.md))
+so it is possible to
+correlate the actions of a data processing server with
+changes to the metadata.
+
+Figure 1 captures how people and teams are represented.
+
+![UML](0110-Actors.svg#pagewidth)
```

Comment:
The "pagewidth" qualification should no longer be there on the image markdown.

Reply:
Done. Updated as per your request.